src/components/FlowList.vue

```diff
@@ -1,236 +1,222 @@
-            <FlowListItem
-              v-model:selected="selected"
-              :value="item.id"
-              :flow="item"
-              :filter="routeFilter"
-              :selectable="selectable"
-              class="flow-list__flow"
-              @update="handleUpdate"
-              @delete="handleDelete"
-            />
-          </slot>
+      <template #header-sort>
+        <p-select v-model="filter.sort" :options="flowSortOptions" />
+      </template>
+
+      <p-table :data="flows" :columns="columns">
+        <template #selection-heading>
+          <p-checkbox v-model="model" @update:model-value="selectAllFlows" />
+        </template>
+
+        <template #selection="{ row }">
+          <p-checkbox v-model="selectedFlows" :value="row.id" />
+        </template>
+
+        <template #name="{ row }">
+          <div class="flow-list__name-col">
+            <p-link :to="routes.flow(row.id)" class="flow-list__name">
+              <span>{{ row.name }}</span>
+            </p-link>
+            <span class="flow-list__created-date">Created {{ formatDateTimeNumeric(row.created) }}</span>
+          </div>
+        </template>
+
+        <template #last-run="{ row }">
+          <LastFlowRun :flow-id="row.id" />
         </template>
-      </p-virtual-scroller>
+
+        <template #next-run="{ row }">
+          <NextFlowRun :flow-id="row.id" />
+        </template>
+
+        <template #deployments="{ row }">
+          <DeploymentsCount :flow-id="row.id" />
+        </template>
+
+        <template #activity="{ row }">
+          <MiniFlowHistory
+            class="flow-list__activity-chart"
+            :flow-id="row.id"
+            :time-span-in-seconds="activityTimeSpanInSeconds"
+          />
+        </template>
+
+        <template #action-heading>
+          <span />
+        </template>
+
+        <template #action="{ row }">
+          <div class="flow-list__action">
+            <FlowMenu size="xs" :flow="row" flat @delete="refresh" />
+          </div>
+        </template>
+
+        <template #empty-state>
+          <PEmptyResults>
+            <template #message>
+              No flows
+            </template>
+            <template v-if="isCustomFilter" #actions>
+              <p-button small @click="clear">
+                Clear Filters
+              </p-button>
+            </template>
+          </PEmptyResults>
+        </template>
+      </p-table>
 
       <template #footer-end>
-        <slot name="footer-end" v-bind="{ page, pages }">
-          <p-pager v-if="pages > 1" v-model:page="page" :pages="pages" />
-        </slot>
+        <p-pager v-if="pages > 1" v-model:page="page" :pages="pages" />
       </template>
     </p-layout-table>
   </div>
 </template>
 
 <script lang="ts" setup>
-  import { media } from '@prefecthq/prefect-design'
+  import { PTable, PEmptyResults, PLink, CheckboxModel, TableColumn } from '@prefecthq/prefect-design'
```

Comment:
Don't import these from prefect design. If you import them it'll always use the prefect-design version set in ui-library rather than using the peer installed in oss/cloud
```suggestion
```

Reply:
Ah ok, copy paste err. I'll still need to import the types though, unless I'm wrong there.

Reply:
Oh right the types are fine. Just not the components